### ISSUE TITLE
Latest Docker image does not seem to work. 

### ISSUE THREAD

**Issue by cdurepo** [meeb/tubesync#291]
The latest docker image seems to have a problem creating the database so it cannot run. 
Looking at the logs: 

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/django/db/backends/base/base.py", line 242, in _commit
  Apply all migrations: admin, auth, background_task, contenttypes, sessions, sync
Running migrations:
    return self.connection.commit()
sqlite3.OperationalError: database is locked

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/django/db/migrations/recorder.py", line 68, in ensure_schema
    editor.create_model(self.Migration)
  File "/usr/local/lib/python3.9/dist-packages/django/db/backends/sqlite3/schema.py", line 36, in __exit__
    super().__exit__(exc_type, exc_value, traceback)
  File "/usr/local/lib/python3.9/dist-packages/django/db/backends/base/schema.py", line 120, in __exit__
    self.atomic.__exit__(exc_type, exc_value, traceback)
  File "/usr/local/lib/python3.9/dist-packages/django/db/transaction.py", line 246, in __exit__
    connection.commit()
  File "/usr/local/lib/python3.9/dist-packages/django/utils/asyncio.py", line 33, in inner
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/django/db/backends/base/base.py", line 266, in commit
    self._commit()
  File "/usr/local/lib/python3.9/dist-packages/django/db/backends/base/base.py", line 242, in _commit
    return self.connection.commit()
  File "/usr/local/lib/python3.9/dist-packages/django/db/utils.py", line 90, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/local/lib/python3.9/dist-packages/django/db/backends/base/base.py", line 242, in _commit
    return self.connection.commit()
django.db.utils.OperationalError: database is locked

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/app/manage.py", line 18, in <module>
    main()
  File "/app/manage.py", line 14, in main
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.9/dist-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.9/dist-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.9/dist-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.9/dist-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.9/dist-packages/django/core/management/base.py", line 89, in wrapped
    res = handle_func(*args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/django/core/management/commands/migrate.py", line 244, in handle
    post_migrate_state = executor.migrate(
  File "/usr/local/lib/python3.9/dist-packages/django/db/migrations/executor.py", line 91, in migrate
    self.recorder.ensure_schema()
  File "/usr/local/lib/python3.9/dist-packages/django/db/migrations/recorder.py", line 70, in ensure_schema
    raise MigrationSchemaMissing("Unable to create the django_migrations table (%s)" % exc)
django.db.migrations.exceptions.MigrationSchemaMissing: Unable to create the django_migrations table (database is locked)
[cont-init.d] 10-tubesync: exited 1.

Then it just keeps repeating: 
Failed to retrieve tasks. Database unreachable.

I have set the workers to one, but that does not seem to fix the lock issue.

**Comment by meeb** [comment]
Is any other process accessing the SQLite database? Or is the SQLite database (the `/config` volume) on a network share?

**Comment by cdurepo** [comment]
There is no other process trying to use the database only the docker image.  yes it is on a network share. Since you are asking I assume that is a problem.  Is there a work around to make it work on a network share or do I just need to move it?

**Comment by cdurepo** [comment]
Okay I did not know that.  Thank you.  
I have moved the config dir to to the local drive and it did start up.  I was able to add channels without issues.  But is seems to not be downloading anything.  It says the download will start at a time that has already passed.  I checked the config dir and I see the cache directory but it does not have any videos.  It was able to create the download folders for each channel I added, it knows there are videos to download but it has not downloaded any yet.  
Is having the video directory on a share an issue?
